src/runtime/pkg/katatestutils/utils.go

```diff
@@ -318,6 +319,7 @@ func MakeRuntimeConfigFileData(config RuntimeConfigOptions) string {
 	msize_9p = ` + strconv.FormatUint(uint64(config.DefaultMsize9p), 10) + `
 	enable_debug = ` + strconv.FormatBool(config.HypervisorDebug) + `
 	guest_hook_path = "` + config.DefaultGuestHookPath + `"
+    guest_hook_timeout = ` + strconv.FormatUint(uint64(config.DefaultGuestHookTimeout), 10) + `
```

Comment:
Please fix the formatting here with gofmt.

Reply:
It was formatted locally, but I'm not sure why it shows differently in Github.
<img width="919" alt="image" src="https://user-images.githubusercontent.com/16031013/187018672-97e6875e-09d0-41c1-b10e-3cc53adcbe16.png">